src/main/java/task/Event.java

```diff
@@ -0,0 +1,57 @@
+package task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import chatbot.DateTimeParser;
+
+/*
+ * Represents an Event Task, which has a start and end time.
+ * Inherits from the Task Class.
+ */
+public class Event extends Task {
+    private LocalDateTime  from;
+    private LocalDateTime  to;
```

Comment:
Consider removing the additional whitespace

```suggestion
    private LocalDateTime from;
    private LocalDateTime to;
```